src/cli/init.rs

```diff
@@ -1,31 +1,36 @@
-use std::io::{self, Write};
-use std::path::{Path, PathBuf};
-use std::process::{Command, Stdio};
+use std::io::{ self, Write };
+use std::path::{ Path, PathBuf };
+use std::process::{ Command, Stdio };
```

Comment:
You seem to have your formatter configured in a way that isn't the rustfmt default; it's causing noise in here and will cause CI to fail.